packages/treats/client/index.js

```diff
@@ -42,7 +42,8 @@ const initClient = params => {
             navigator.language ||
             navigator.userLanguage ||
             "en-US",
-        guardedRootDiv = rootDiv || "treats-root";
+        guardedRootDiv = rootDiv || "treats-root",
+        swScript = document.querySelectorAll("script[treats-sw-script]");
```

Comment:
Hi, please make it into data attribute (i.e. `data-treats-sw-script`) so it would pass W3C validator